spel/scripts/amigen7-build.sh

```diff
@@ -267,13 +268,18 @@ function ComposeChrootCliString {
 
 }
 
-set -x
```

Comment:
These `set` options are still important, I think they should remain. They should be added to the amigen8 script, but not in this PR because it may require other adjustments.

Reply:
So my thought on the `set` options was that in amigen8 they are used every time a script is actually invoked example `   bash -euxo pipefail "${ELBUILD}"/$( ComposeChrootMountString ) || \
     err_exit "Failure encountered with MkChrootTree.sh"`

Reply:
That only applies to those scripts though, not *this* script. Any command failure in this script will be ignored, which can lead to a false success or to misleading errors in the log as the script continues to run.

Reply:
Put the set bulletins just before the execution of the main script steps